.github/workflows/python-publish.yml

```diff
@@ -49,7 +46,9 @@ jobs:
         run: |
           # NOTE: put your own distribution build steps here.
           python -m pip install build
-          python -m build
+          export SETUPTOOLS_SCM_PRETEND_VERSION=${GITHUB_REF}
+          export SETUPTOOLS_SCM_DEBUG=1
+          python -m build --sdist --wheel ./
```

Comment:
https://github.com/pypa/setuptools-scm/issues/997#issuecomment-2613474797

Was seeing this "error" too...